pallene/translator.lua

```diff
@@ -141,6 +216,8 @@ end
 function translator.translate(input, prog_ast)
     local instance = Translator.new(input)
 
+    -- print(require('inspect')(prog_ast))
+
```

Comment:
Another print statement left behind. More than one today ;)

Reply:
Lots to debug :(